extensions/src/ACRE2Arma/common/glm/detail/setup.hpp

```diff
@@ -653,7 +653,7 @@
 #endif
 
 // OpenMP
-#ifdef _OPENMP 
+#ifdef _OPENMP
```

Comment:
Why are all these space at end of line changes part of this PR, they have nothing to do with this

Reply:
VS Code is removing them when I save a file

Reply:
That's fine, need to clean that up anyways.